mw-webapp/src/logic/wayPage/reportsTable/reportsColumns/reportsTablePlansCell/ReportsTablePlansCell.tsx

```diff
@@ -312,6 +312,7 @@ export const ReportsTablePlansCell = observer((props: ReportsTablePlansCellProps
                   <Modal
                     trigger={
                       <Checkbox
+                        isDisabled={true}
```

Comment:
Should it be always disabled?

Reply:
Yes, by default isDisabled set to false and the checkbox triggers (changes its value) every time it's clicked. However, if we explicitly set it to true, we can control the click on it

Reply:
We disable fuction handleCheckboxChange
```
  const handleCheckboxChange = () => {
    setIsChecked(!isChecked);
    props.onChange(!isChecked);
  };
```
and the checkbox will only trigger when props.isDefaultChecked changes